src/contracts/proxies/actions/BasicActions.sol

```diff
@@ -398,4 +398,26 @@ contract BasicActions is CommonActions, IBasicActions {
     // Transfers token amount to the user's address
     _collectAndExitCollateral(_manager, _collateralJoin, _safeId, _collateralWad);
   }
+
+  /// @inheritdoc IBasicActions
+  function collectSystemCoins(
+    address _manager,
+    address _coinJoin,
+    uint256 _safeId,
+    uint256 _deltaWad
+  ) external onlyDelegateCall {
+    // Transfers COIN amount to the user's address
+    _collectAndExitCoins(_manager, _coinJoin, _safeId, _deltaWad);
+  }
```

Comment:
not needed, we wanted to do minimal work here, but ok i'd say, i don't see a possibility where the safeHandler would get some coins tbh

Reply:
I added it just in case. I think it's good for increasing fault tolerance.